jobsdb/jobsdb.go

```diff
@@ -148,6 +149,26 @@ var dbErrorMap = map[string]string{
 	"Invalid Escape Character": "22019",
 }
 
+// return backup settings depending on jobdb type
+func getBackUpSettings(jd *HandleT) *BackupSettingsT {
+	var pathPrefix string
+	config.Initialize()
+	// for replay server, make the instance (gw/rt/batch_rt) backup accordingly in the config file itself
+	masterBackupEnabled := config.GetBool("JobsDB.backup.enabled", false)
+	instanceBackupEnabled := config.GetBool(fmt.Sprintf("JobsDB.backup.%v.enabled", jd.tablePrefix), false)
+	instanceBackupFailedAndAborted := config.GetBool(fmt.Sprintf("JobsDB.backup.%v.abortedOnly", jd.tablePrefix), false)
+	if jd.tablePrefix == "gw" {
```

Comment:
RIght now - jobsdb is not aware of the table prefix values (gw/rt/batch_rt) etc. 
So instead of setting a hard check here - Let's put this also in the config? 

Default path prefix can be the tablePrefix, we can override it for gateway by setting to empty string.
```
[JobsDB.backup.gw]
enabled = true
pathPrefix=""
```

Reply:
done